packages/language-server/lib/server.js

```diff
@@ -22,7 +23,22 @@ connection.onInitialize(() => {
 documents.onDidChangeContent(async (event) => {
   const diagnostics = await validateDocument(event.document);
   connection.sendDiagnostics({ uri: event.document.uri, diagnostics });
+
+  if (diagnostics.length > 0) {
+    // The language server (currently) only exposes **syntactic** XML diagnostics.
+    getSWA().track("Syntax Issues Detected", [{ number: diagnostics.length }]);
```

Comment:
Does the swa library adds a prefix or some other way to identify the extension name? If not you should add it

Reply:
Yeah it add a prefix constructed from the `publisher` and `extName` passed to it during the constructor.
This prefix is added to all `track` calls.